src/nxt_status.c

```diff
@@ -9,14 +9,22 @@
 
 
 nxt_conf_value_t *
-nxt_status_get(nxt_status_report_t *report, nxt_mp_t *mp)
+nxt_status_get(nxt_task_t *task, nxt_status_report_t *report, nxt_mp_t *mp)
 {
     size_t            i;
     nxt_str_t         name;
+    nxt_str_t         version;
     nxt_int_t         ret;
     nxt_status_app_t  *app;
     nxt_conf_value_t  *status, *obj, *apps, *app_obj;
+    nxt_runtime_t     *rt;
 
+    rt = task->thread->runtime;
+
+    static nxt_str_t unit_str = nxt_string("unit");
+    static nxt_str_t ver_str = nxt_string("version");
+    static nxt_str_t gen_str = nxt_string("generation");
+    static nxt_str_t ltime_str = nxt_string("load_time");
     static nxt_str_t conns_str = nxt_string("connections");
     static nxt_str_t acc_str = nxt_string("accepted");
     static nxt_str_t active_str = nxt_string("active");
```

Comment:
Variable declarations should come before code... (I know there are some places that break this).